fintech-examples/fintech-ui/src/app/interceptors/auth.interceptor.ts

```diff
@@ -30,6 +33,9 @@ export class AuthInterceptor implements HttpInterceptor {
         headers
       });
     }
+    for (const key of request.headers.keys()) {
+      console.log('new request has header key', key, ' with value: ', request.headers.get(key));
+    }
```

Comment:
same here --> let's probably not commit it. We can inspect the headers anyways when we send a request